web-registry/src/components/organisms/CreditBatches.tsx

```diff
@@ -5,7 +5,7 @@ import TableBody from '@mui/material/TableBody';
 import TableHead from '@mui/material/TableHead';
 import TableRow from '@mui/material/TableRow';
 import TableFooter from '@mui/material/TableFooter';
-import moment from 'moment';
+import { format } from 'date-fns';
```

Comment:
does this have a "2 weeks ago" format? I like moment for all of its formatting options, but I haven't used `date-fns`

Reply:
@mhagel Yes it does! [formatRelative](https://date-fns.org/v2.28.0/docs/formatRelative)

Moment has pretty much all of the same methods as moment, but a slightly different API. Again, I'm fine switching this to day.js (which has the exact same API as moment) if ya'll would prefer - I don't really have much of a preference other than not using moment. 

I'd say if you're already familiar with moment's api, it might make sense to just use day.js - I can update this PR accordingly

Reply:
I'm good with whatever. If we can reduce bundle size by avoiding moment, I'm all for it.